README.md

```diff
@@ -13,6 +13,7 @@ go get github.com/heimweh/go-pagerduty/pagerduty
 ```
 
 ## Example usage
+1. Create the following in a file at some path like the project root `go-pagerduty/main.go`
```

Comment:
I suggest you to better recommend to follow a [Go Modules](https://go.dev/doc/modules/managing-dependencies#adding_dependency) approach, because this is a library that can be used independently of the Terraform Provider following the next steps to kickstart a project...

```sh
$ go mod init go-pagerduty-example
$ go get github.com/heimweh/go-pagerduty
$ go mod tidy
```

Reply:
As you originally suggested to me: "Anyone contributing to this should already know how to use a go file"

Since these directions and suggestions could change even the links you've provided could change, I think it's better to just leave out any recommendations on how to use the file.